Cloudinary/Api.cs

```diff
@@ -77,7 +77,7 @@ static Api()
         /// <param name="parameters">Dictionary of call parameters (can be null)</param>
         /// <param name="file">File to upload (must be null for non-uploading actions)</param>
         /// <returns>HTTP response on call</returns>
-        public HttpWebResponse Call(HttpMethod method, string url, SortedDictionary<string, object> parameters, FileDescription file, Dictionary<string, string> extraHeaders = null)
+        public override object Call(HttpMethod method, string url, SortedDictionary<string, object> parameters, FileDescription file, Dictionary<string, string> extraHeaders = null)
```

Comment:
I clearly understand the need for cast.
My point is that since this method is public, if someone called it directly (someone == SDK user), she will need to change her code (adding casting)